pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -72,6 +75,7 @@ public void send(TransferableBlock block)
   @Override
   public void complete()
       throws Exception {
+    // TODO: should wait for _mailboxContentStreamObserver.onNext() finish before calling onComplete().
```

Comment:
currently it is guaranteed that onComplete is called after all onNext() --> implicitly via the OpChain scheduler